### DISCUSSION TITLE
Favicon handling

### DISCUSSION THREAD

**Discussion by AntoineGlacet** [mmistakes/minimal-mistakes#3447] [Q&A] [answered]
Hello,
I do not understand how is the favicon handled in the theme. Mind you, it works fine when I put an icon at the root of my site folder, however I also want to add ios-touch and android icon. I want to keep a consistent structure so I want to add these in the same way as favicon is added but I cannot find where is the reference to favicon added to my pages. I can't find it anywhere in the html files served by jekyll!

**Comment by iBug** [comment, marked as answer]
You can provide your own `_includes/head/custom.html` and insert any favicons you need.

See <https://mmistakes.github.io/minimal-mistakes/docs/layouts/#head> and <https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/head/custom.html>.

**Reply by AntoineGlacet** [reply]
great thank you, I actually thought it was already done somewhere as I did not realize browsers just look for favicon.ico in root without any html pointer. However, for other stuff like android or apple touch we need to put some more references in the head as you mentioned.

Thanks!
